test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using AutoRest.TestServer.Tests.Infrastructure;
+using Azure;
+using Azure.Core;
+using dpg_customization_LowLevel;
+using dpg_customization_LowLevel.Models;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class DpgCustomizationTest : TestServerLowLevelTestBase
```

Comment:
Are the LRO grow-up methods in this group of tests?

Reply:
No, these will be covered after @AlexanderSher finished his refactor on LRO.

Reply:
Is there a separate issue tracking this additional work?  It would be good to link that from here.

Reply:
I believe it is https://github.com/Azure/autorest.csharp/issues/1706. @AlexanderSher please correct me if I'm wrong.

Reply:
Thanks, @pshao25!  Actually, I meant do we have an issue tracking adding the LRO test in our DPG sprint.  It would be great to have one so we can make sure we know there are outstanding tests to write when we close https://github.com/azure/autorest.csharp/issues/1944

Reply:
Tracked by #2027